pennylane/estimator/__init__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This module contains tools for logical resource estimation.
+
+.. currentmodule:: pennylane.labs.estimator
```

Comment:
```suggestion
.. currentmodule:: pennylane.estimator
```
Is the decision from product to rename `resource_estimation` to `estimator`?

Reply:
yes, this was product's decision.